src/CTA.Rules.Actions/ActionHelpers/ConfigMigrate.cs

```diff
@@ -22,6 +22,15 @@ public ConfigMigrate(string projectDir, ProjectType projectType)
             _projectType = projectType;
             _hasData = false;
         }
+
+        public string WebformsWebConfigMigrateHelper()
+        {
+            Configuration configXml = LoadWebConfig(_projectDir);
+            var templateContent = @"{}";
```

Comment:
Let's replace templateContent with:

```csharp
var templateContent = TemplateHelper.GetTemplateFileContent(string.Empty, ProjectType.WebClassLibrary, Constants.AppSettingsJson);
```

What this will do is load an appsettings.json template file into templateContent- this will keep us in line with the existing behavior for other project types. This template is downloaded from our S3 templates bucket when a `SolutionPort` object is instantiated.

Reply:
I'm having an error when switching templateContent, saying there is no appsettings.json template to load. Do I have to create the template myself?

Reply:
Per our conversation, the issue was a pathing error which we were able to resolve.